### PR TITLE
fix: move icons to gressource

### PR DESCRIPTION
**Pull request by 0xMRTT**
Icons are currently installed in datadir, it can cause conflict with others apps (maybe they are using the same icons). So, i move icons to  `gressource`

### CODE REVIEW

**Review by mjakeman — CHANGES_REQUESTED** [review]
Thanks for working on this!

The change looks good to me - makes sense to bundle them as resources.

It seems like you've changed the formatting for `exm-rating.c`. I'm not sure if there is any change here, or if it's just been formatted by accident, but please stick to the project's coding style.

**Comment by 0xMRTT** [comment]
Oh sorry, it's because I thought that I need to change something but i was wrong

**Comment by 0xMRTT** [comment]
Fixed

File: src/exm-rating.c [hunk 1]
```diff
@@ -143,4 +143,4 @@ exm_rating_init (ExmRating *self)
     gtk_widget_init_template (GTK_WIDGET (self));
 
     update_rating (self);
-}
+}
```
Comment (mjakeman) [inline src/exm-rating.c:146]: Please don't remove the newline here as it pollutes git history. You can remove this file altogether using:

```
# Undo last commit(s) where 1 is the number of commits to undo
git reset --soft HEAD~1

# Add only the relevant files
git add src/exm.gresource.xml
git add data

# Recommit
git commit -m "..."
```

File: src/exm.gresource.xml [hunk 2]
```diff
@@ -16,4 +16,19 @@
     <file>release-notes.txt</file>
     <file>style.css</file>
   </gresource>
-</gresources>
+
+  <gresource prefix="/com/github/GradienceTeam/Gradience/icons/scalable/actions/">
```
Comment (mjakeman) [inline src/exm.gresource.xml:20]: Wrong prefix :)

Should be `/com/mattjakeman/ExtensionManager/...`

**Review by mjakeman — CHANGES_REQUESTED** [review]
A few other small issues I noticed

Once fixed, could you squash this into a single commit please to keep git history tidy?

**Review by mjakeman — APPROVED** [review]

**Comment by mjakeman** [comment]
Let's get this merged for 0.4, thanks for the contribution!

**Comment by 0xMRTT** [comment]
Thanks

**Pull request merged**